core/amber/src/main/protobuf/edu/uci/ics/texera/workflowruntimestate.proto

```diff
@@ -74,6 +79,7 @@ message OperatorRuntimeStats{
 
 message JobStatsStore {
   map<string, OperatorRuntimeStats> operator_info = 1;
+  repeated OperatorWorkerMapping operator_worker_mapping = 2;
```

Comment:
why don't we use a map from operatorID to an array of worker ids, similar to the line above?

Reply:
protobuf does not support map of array. it has to explicitly define a wrapper message like 
`message WorkerIdList {
    repeated string workerIds = 1;
}
message JobStatsStore {
  map<string, OperatorRuntimeStats> operator_info = 1;
  map<string, WorkerIdList> operator_worker_mapping= 2;
}
`
then to access it you need to use `jobStatsStore.operator_worker_mapping(opId).workerIdList.workerIds` which is lengthy..

Here this version is considered to be better.. I can use `jobStatsStore.operator_worker_mapping(opId).workerIds`.